lightly/utils/bounding_box.py

```diff
@@ -77,24 +80,49 @@ def clip_to_0_1(value: float) -> float:
 
     @classmethod
     def from_x_y_w_h(cls, x: float, y: float, w: float, h: float) -> BoundingBox:
-        """Helper to convert from bounding box format with width and height.
+        """Creates a BoundingBox from x, y, width, and height.
+
+        Args:
+            x: 
+                x coordinate of the top-left corner.
```

Comment:
Again: Let's document that the values are relative to the image width/height and not in the unit of pixels.
```suggestion
                x coordinate of the top-left corner. Relative to the image width.
```